packages/clerk-js/src/ui/components/OrganizationProfile/ActionConfirmationPage.tsx

```diff
@@ -108,19 +131,31 @@ const ActionConfirmationPage = withCardStateProvider((props: ActionConfirmationP
         Breadcrumbs={OrganizationProfileBreadcrumbs}
       >
         <Form.Root onSubmit={handleSubmit}>
-          <Text
-            localizationKey={messageLine1}
-            variant='regularRegular'
-          />
-          <Text
-            localizationKey={messageLine2}
-            variant='regularRegular'
-          />
+          <Text localizationKey={messageLine1} />
+          <Text localizationKey={messageLine2} />
+
+          {actionDescription && (
+            <>
+              <Text
+                localizationKey={actionDescription}
+                variant='regularRegular'
```

Comment:
If this variant is the default one, maybe skip it. Correlates to ☝️

Reply:
@panteliselef i should remove this as well